infrastructure/helm-chart/charts/core/charts/upgrading/templates/0.29.0.yaml

```diff
@@ -45,36 +45,6 @@ data:
       curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer ${system_token}" -d "${request}" api-admin/webhooks.subscribe
     fi
 ---
-# Cleanup old jobs
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: {{ .Values.global.kubernetes.appImageTag }}-cleanup-upgrade
-  namespace: {{ .Values.global.kubernetes.namespace }}
-  annotations:
-    "helm.sh/hook": "pre-upgrade"
-    "helm.sh/hook-weight": "0"
-  labels:
-    core.airy.co/upgrade: "true"
-spec:
-  ttlSecondsAfterFinished: 120
-  template:
-    spec:
-      containers:
-      - name: reset-app
-        image: "bitnami/kubectl"
-        command: ['/bin/sh', '/opt/provisioning/upgrade-cleanup.sh']
-        volumeMounts:
-        - name: upgrading-scripts
-          mountPath: /opt/provisioning
-      serviceAccountName: airy-upgrade
-      volumes:
-        - name: upgrading-scripts
-          configMap:
-            name: upgrading-scripts
-      restartPolicy: Never
-  backoffLimit: 3
----
```

Comment:
Moved to a generic yaml file, so that the cleanup is executed on every run of `airy upgrade`.